notification/store.go

```diff
@@ -256,51 +243,42 @@ func (db *DB) SendContactMethodVerification(ctx context.Context, id string, rese
 		return validation.NewFieldError("ContactMethod", "test message rate-limit exceeded")
```

Comment:
We should probably change this to be something like `rate limit exceeded` or something more specific to codes and not test messages

Reply:
for sure

Reply:
We could probably have the UI display a nicer message if we get `rate limit exceeded` too. We haven't really solved structured errors yet though.

It would be nice to let the user know they should try again after a few seconds (or a minute, whatever we have it set to).